docs/release_notes/june_2024.md

```diff
@@ -0,0 +1,36 @@
+# June 2024
+
+## airbyte v0.62.3 to v0.63.4
+
+This page includes new features and improvements to the Airbyte Cloud and Airbyte Open Source platforms.
+
+## ✨ Highlights
+
+Airbyte added [Refresh](operator-guides/refreshes) support for our BigQuery destination, which brings an improved experience to resyncing all your data again. This enables data to never be deleted from final tables during a historical resyncing of data. More certified destinations will be supported in the coming weeks. 
+
+Users can now monitor the progress of syncs through the addition of a live counter on the Connection Status page. The sync progress feature displays record counts and duration for each stream during a sync, which streams are syncing, and when a sync is not running. It also shows counts from the last time each enabled stream synced. Lastly, each stream now shows an individual stream status, which helps you troubleshoot by allowing you to pinpoint exactly which streams are erroring.
+
+![Sync Progress](./assets/sync-progress.png)
+
+## Platform Releases
+
+- (Cloud, Cloud Teams, and Self-Managed Enterprise only) Connections now display both the stream status and the latest records moved for recent syncs to aid with troubleshooting. 
+
+![Connection Graph](./assets/connection-stream-status-graph.png)
+
+- (Cloud Teams and Self-Managed Enterprise only) New roles have been released to support more defined access for larger teams. Editors (for Workspaces) and Readers (both Organization and Workspace) are now available. Read more in our [docs](/access-management/rbac).
+
+## Connector Improvements
+
+We also released a few notable improvements for our connectors:
+
+- In the Connector Builder, `Undo` and `Redo` have been added, which can also be accessed through cmd-z / cmd-shift-z hotkeys. This allows users to test without fear of losing their previous state within the same session. The Connector Builder is also now a full-screen experience with an improved header menu. Additionally, the `Test` button can now also be triggered by pressing cmd-enter, so users no longer need to manually click on `Test` while actively developing.
+
+- PyAirbyte now runs YAML sources with no additional installation needed, opening up more ways to sync data.
+
+- With CDK version 1.5.0, all low-code connectors’ will have a standardized way of reporting errors. The CDK automatically categorizes errors as config, system, or transient based on a default error mapping.
+
+- Our connector docs also now contain Connector Quality Metrics that help users understand the health of a connector through the percentage of successful syncs and how actively used it is. We’ve also added an indication of how recently the connector has been updated.
+
+## Announcements
+We announced we will formally deprecate support for Docker Compose deployments in favor of Kubernetes, and welcome beta testers [here](https://github.com/airbytehq/airbyte/discussions/40599) who want migration assistance. 
```

Comment:
@nataliekwong, could you also include a request for feedback about the API endpoint discussion too?

Reply:
Added!